tornado-drivers/opencl/src/main/java/uk/ac/manchester/tornado/drivers/opencl/enums/OCLBuildStatus.java

```diff
@@ -24,35 +24,30 @@
 package uk.ac.manchester.tornado.drivers.opencl.enums;
 
 public enum OCLBuildStatus {
-    CL_BUILD_SUCCESS(0), CL_BUILD_NONE(-1), CL_BUILD_ERROR(-2), CL_BUILD_IN_PROGRESS(-3), CL_BUILD_UNKNOWN(-4);
+    CL_BUILD_SUCCESS(0), //
+    CL_BUILD_NONE(-1), //
+    CL_BUILD_ERROR(-2), //
+    CL_BUILD_IN_PROGRESS(-3), //
+    CL_BUILD_UNKNOWN(-4);
 
-    private final int value;
+    private final int errorCode;
```

Comment:
Is the name `errorCode` appropriate? The build might be successful. How about something like `buildCode`?

Reply:
Yes, we can rename the variable name to `buildCode` to include also the successful build code.